src/libs/actions/Transaction.ts

```diff
@@ -73,6 +73,7 @@ function saveWaypoint(transactionID: string, index: string, waypoint: RecentWayp
         // Clear the existing route so that we don't show an old route
```

Comment:
Actually sorry, can we update this comment or add a new one nearby explaining why we're clearing out `distance`, just so it's easy to understand / remember why this is needed in the future? 🙏

Reply:
Nice, thanks @dukenv0307 - do you mind adding the comment to the `distance: null` you added as well? Seems you added the comment to another place we set distance to null, which is also useful!

Reply:
Haha we must clear distance a lot -> You added `distance: null` on line 76, but added comments on line 150 & 260 🤣

Reply:
No worries! Apparently we clear distance in many places hahaha now we both know!